dask_ml/preprocessing/data.py

```diff
@@ -915,3 +915,62 @@ def inverse_transform(self, X):
             X[col] = series
 
         return X
+
+
+class PolynomialFeatures(skdata.PolynomialFeatures):
+
+    __doc__ = skdata.PolynomialFeatures.__doc__
+
+    def __init_(self, degree=2, interaction_only=False, include_bias=True):
+        super(PolynomialFeatures, self).__init__(degree, interaction_only, include_bias)
+
+    def fit(self, X, y=None):
+        """
+        Compute number of output features.
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            The data.
+        Returns
+        -------
+        self : instance
+
+        This implementation was copied and modified from Scikit-Learn.
+
+        See License information here:
+        https://github.com/scikit-learn/scikit-learn/blob/master/README.rst
+        """
+        self._transformer = skdata.PolynomialFeatures()
+        X_sample = X
+        if isinstance(X, dd.DataFrame):
+            X_sample = X._meta
+        if isinstance(X, da.Array):
+            X_sample = np.ones((1, X.shape[1]), dtype=X.dtype)
+
+        self._transformer.fit(X_sample)
```

Comment:
After we fit, we'll want to copy over the attribute learned attributes. I think that `dask_ml._utils.copy_learned_attributes(self._transformer, self)` will work here.